### DISCUSSION TITLE
Multiple Recipients

### DISCUSSION THREAD

**Discussion by ThomasMichel92** [globaleaks/globaleaks-whistleblowing-software#3726] [General]
Hello Globaleaks Developer Team,

i have a question on how to work with multiple recipients.

At the Moment only the first recipient we have created gets a mail notification if a new report was created.
We have 3 Recipients activated. So that if there is someone on vacation another one could work with the report.

Well 2 of the 3 dont get any mail notifications unless the first recipient share the report with them.

Is there a way to set up the usecase which I have described?

King Regards!

**Comment by evilaliv3** [comment]
@ThomasMichel92 : i think you may have created users but you may have forgot to add them as recipient for your channel.

Get sure to add your additional recipients to the reporting channel.
